miscelanea/004-route-traffic/poc-004-2.yaml

```diff
@@ -0,0 +1,204 @@
+# https://kubernetes.io/docs/concepts/services-networking/service/
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: poc-004-2-client
+  labels:
+    app: poc-004-2
+    role: client
+spec:
+  affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - topologyKey: "kubernetes.io/hostname"
+          labelSelector:
+            matchLabels:
+              app: poc-004-2
+              role: services
```

Comment:
If this block is replicated into the poc-004-2-services, they pods were not never scheduled; it was permanent into the scheduling state; looks like the condition never were satisfied.

Reply:
I have added a note into the yaml file